applications/luci-app-v2raya/htdocs/luci-static/resources/view/v2raya/config.js

```diff
@@ -0,0 +1,110 @@
+'use strict';
+'require form';
+'require poll';
+'require rpc';
+'require uci';
+'require view';
+
+var callServiceList = rpc.declare({
+	object: 'service',
+	method: 'list',
+	params: ['name'],
+	expect: { '': {} }
+});
+
+function getServiceStatus() {
+	return L.resolveDefault(callServiceList('v2raya'), {}).then(function (res) {
+		var isRunning = false;
+		try {
+			isRunning = res['v2raya']['instances']['v2raya']['running'];
+		} catch (e) { }
+		return isRunning;
+	});
+}
+
+function renderStatus(isRunning, port) {
+	var spanTemp = '<span style="color:%s"><strong>%s %s</strong></span>';
+	var renderHTML;
+	if (isRunning) {
+		var button = String.format('&#160;<a class="btn cbi-button" href="http://%s:%s" target="_blank" rel="noreferrer noopener">%s</a>',
+			window.location.hostname, port, _('Open Web Interface'));
+		renderHTML = spanTemp.format('green', _('v2rayA'), _('RUNNING')) + button;
+	} else {
+		renderHTML = spanTemp.format('red', _('v2rayA'), _('NOT RUNNING'));
+	}
+
+	return renderHTML;
+}
+
+return view.extend({
+	load: function() {
+		return Promise.all([
+			uci.load('v2raya')
+		]);
+	},
+
+	render: function(data) {
+		var m, s, o;
+		var webport = (uci.get(data[0], 'config', 'address') || '0.0.0.0:2017').split(':').slice(-1)[0];
+
+		m = new form.Map('v2raya', _('v2rayA'),
```

Comment:
Is the app called v2raya? This seriously needs disambiguating from the dead `v2ray` client, versus the actively developed `v2fly` project. Perhaps a web link to the github page or (Englilsh) webpage? Zh i18n can have the link to the Zh version.

Reply:
> Is the app called v2raya?

Yes.

> This seriously needs disambiguating from the dead `v2ray` client, versus the actively developed `v2fly` project

Strictly say, `v2ray` client is not dead.

The v2ray organization on GitHub was deprecated because of the lack of admin access (the original author (owner) is no longer active). (This is the statement from active maintainers.)

Active maintainers created a new organization called `v2fly`, but did not rename the core, it's still `v2ray-core`.